File: Swift/TextView/TextView/ViewController.m
//
//  ViewController.m
//  TextView
//
//  Created by lyf on 2023/8/17.
//

#import "ViewController.h"

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    // Do any additional setup after loading the view.
}

- (void)viewDidAppear {
    [super viewDidAppear];
    [self.webview loadRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:@"https://cn.bing.com"]]];
}

- (void)setRepresentedObject:(id)representedObject {
    [super setRepresentedObject:representedObject];

    // Update the view, if already loaded.
}


@end

File: appKey/appKey/ViewController.swift
//
//  ViewController.swift
//  appKey
//
//  Created by lyf on 2023/7/6.
//

import Cocoa
import Carbon

class ViewController: NSViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        var spec = EventTypeSpec()
        InstallEventHandler(GetApplicationEventTarget(), { handle, event, userData in
            var hkCom = EventHotKeyID()
            return GetEventParameter(event, EventParamName(kEventParamDirectObject), EventParamType(typeEventHotKeyID), nil, MemoryLayout<EventTypeSpec>.size, nil, &hkCom);
            
            
        }, 1, &spec, nil, nil)
        // Do any additional setup after loading the view.
    }

    override var representedObject: Any? {
        didSet {
        // Update the view, if already loaded.
        }
    }


}

